netsim/ansible/templates/bgp/srlinux.cli.j2

```diff
@@ -43,7 +43,7 @@ send-community large {{ 'true' if 'large' in list else 'false' }}
 #}
 {% if g=='ibgp' and bgp.rr|default(0)|bool %}
  route-reflector {
-  cluster-id {{ bgp.rr_cluster_id }}
+  cluster-id {{ bgp.rr_cluster_id|default(False) or bgp.router_id }}
```

Comment:
I would suggest
```
bgp.rr_cluster_id|default( bgp.router_id )
```

Reply:
Thanks for the feedback! That's what I had initially ;)

However, the only way to say "don't calculate cluster ID" is to set **bgp.rr_cluster_id** to **False**, in which case it is **defined** and so the **default** filter does nothing.

I'm guessing `bgp.rr_cluster_id or bgp.router_id` would also work, but with the longer version we're also handling an (impossible?) edge case of "there is no **bgp.rr_cluster_id** attribute"